scipy/optimize/optimize.py

```diff
@@ -2003,12 +1865,18 @@ def _minimize_scalar_brent(func, brack=None, args=(),
     if tol < 0:
         raise ValueError('tolerance should be >= 0, got %r' % tol)
 
-    brent = Brent(func=func, args=args, tol=tol,
-                  full_output=True, maxiter=maxiter)
-    brent.set_bracket(brack)
-    brent.optimize()
-    x, fval, nit, nfev = brent.get_result(full_output=True)
-    return OptimizeResult(fun=fval, x=x, nit=nit, nfev=nfev,
+    a, _, b, _, _, _, nfev = _get_bracket_info(func, args, brack)
```

Comment:
Is it necessary to throw away the middle point and all three function evaluations? Perhaps optional arguments to find_minimum would allow these to be specified in the common case that they are known.